comment_scan/chatter.rb

```diff
@@ -0,0 +1,122 @@
+require "chatx"
+require "htmlentities"
+
+class Chatter
+    attr_reader :HQroom, :rooms
+
+    def initialize(chatXuser, chatXpwd, hqroom, *rooms)
+        if ENV['SHORT_LOGS']
```

Comment:
Why has this logger conditional moved out to `chatter.rb`?

Reply:
It looked like the only place we were using that logger was for the ChatBot. And the only one who knows about ChatBot is chatter. So I figured that was the place to be.

Thinking about it now though, it'd probably be better if we pulled it out and then used it everywhere instead of all the `puts`.